frontend/src/components/viewers/MetricsVisualizations.tsx

```diff
@@ -289,3 +329,42 @@ function buildConfusionMatrixConfig(
     },
   ];
 }
+
+interface ScalarMetricsSectionProps {
+  artifact: Artifact;
+}
+function ScalarMetricsSection({ artifact }: ScalarMetricsSectionProps) {
+  const customProperties = artifact.getCustomPropertiesMap();
+  const name = customProperties.get('name')?.getStringValue();
+  const data = customProperties
+    .getEntryList()
+    .map(x => ({
+      key: x[0],
+      value: JSON.stringify(getMetadataValue(customProperties.get(x[0]))),
+    }))
+    .filter(metric => metric.key !== 'name');
+
+  if (data.length === 0) {
+    return <></>;
```

Comment:
you can also just:
```
return null;
```

but there is no difference

Reply:
Sounds good, updated!